pibooth/camera/opencv.py

```diff
@@ -60,8 +60,8 @@ def __init__(self,
         self._overlay_alpha = 255
 
         self._cam = cv2.VideoCapture(0)
-        self._cam.set(cv2.CAP_PROP_FRAME_WIDTH, self.resolution[0])
-        self._cam.set(cv2.CAP_PROP_FRAME_HEIGHT, self.resolution[1])
+        self.preview_resolution = (self._cam.get(cv2.CAP_PROP_FRAME_WIDTH), self._cam.get(cv2.CAP_PROP_FRAME_HEIGHT))
+        LOGGER.debug("Preview resolution is {0}".format(self.preview_resolution))
```

Comment:
Use %s in logging library:

LOGGER.debug("Preview resolution is %s", self.preview_resolution)